dictionary_amd64.s

```diff
@@ -814,6 +887,49 @@ indexOutOfBounds:
     MOVQ $errnoIndexOutOfBounds, AX
     JMP return
 
+// This is the same algorithm as dictionaryLookupFixedLenByteArrayString but we
+// only store the pointer to the location hodling the value instead of storing
```

Comment:
```suggestion
// only store the pointer to the location holding the value instead of storing
```

Too much Bitcoin recently?

Reply:
Ahah!

The internet will be the end of me!